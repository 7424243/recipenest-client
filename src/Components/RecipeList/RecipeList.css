ul {
    list-style-type: none;
}

.recipe-list-container {
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
    justify-content: space-evenly;
}