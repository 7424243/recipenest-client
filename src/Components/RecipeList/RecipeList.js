import React, {Component} from 'react'
import Header from '../Header/Header'

class RecipeList extends Component {
    render() {
        return (
            <>
                <Header />
                <p>Recipe List goes here...</p>
            </>
        )
    }
}

export default RecipeList