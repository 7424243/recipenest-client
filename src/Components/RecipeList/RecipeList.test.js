import React from 'react'
import ReactDOM from 'react-dom'
import {BrowserRouter} from 'react-router-dom'
import renderer from 'react-test-renderer'
import RecipeList from './RecipeList';

describe('RecipeList Component', () => {

  //smoke test
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><RecipeList/></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  //snapshot test
  it('renders the UI as expected', () => {
    const tree = renderer 
      .create(<BrowserRouter><RecipeList/></BrowserRouter>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
  
})
