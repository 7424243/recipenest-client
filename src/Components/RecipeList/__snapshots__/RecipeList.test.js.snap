// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeList Component renders the UI as expected 1`] = `
<div>
  <ul
    className="recipe-list"
  />
  <div
    className="container"
  />
  <div
    className="pagination"
  >
    <p>
      Page 1 ... 
      <button>
        &gt;&gt;
      </button>
    </p>
  </div>
</div>
`;
